omnigibson/prims/prim_base.py

```diff
@@ -178,7 +178,7 @@ def dump_state(self, serialized=False):
                 - encoded + serialized, 1D numerical np.array capturing this object's state
         """
         assert self._initialized, "Prim must be initialized before dumping state!"
-        return self._dump_state(serialized=serialized)
+        return self.dump_state(serialized=serialized)
```

Comment:
this should be `super`, not `self`

Reply:
Fixed. Sorry for the double whammy here.